app/models/manageiq/providers/lenovo/physical_infra_manager/parser/component_parser.rb

```diff
@@ -36,5 +36,11 @@ def self.parse(source, dictionary)
       end
       result
     end
+
+    def self.get_location_led_info(leds)
+      return nil, nil if leds.blank?
```

Comment:
Request:
/s `nil, nil` to  `return if leds.blank?`

Reply:
Good point, thanks :+1: